package.json

```diff
@@ -1,11 +1,18 @@
 {
   "scripts": {
     "test": "xo && ava",
+    "build": "cd src && each '{*.js,!(libs)/**/*.js}' --parallel 10 -- rollup --config node:webext $PWD/%p --output $PWD/../extension/%p",
```

Comment:
Have a look at 559b71fb78c223dd9a89d0fc26f4f521af252afd for the "clean" alternative